src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
```

Comment:
```suggestion
public class ColorSupplier {
    private Random random = new Random();

    public String getRandomColor() {
```

давай вынесем переменную Random на уровень класса, что б не создавать каждый раз новый экземпляр, когда мы вызываем метод `getRandomColor()`